civihr_default_theme/template.php

```diff
@@ -449,6 +449,9 @@ function _is_hrreports_current_path() {
  */
 function civihr_default_theme_js_alter(&$javascript) {
 
+  $path = drupal_get_path('theme',$GLOBALS['theme']);;
+  drupal_add_js(array('civihr_default_theme' => array('path' => $path)), 'setting');
```

Comment:
Not so sure what's happening here, this this is in a `js_alter` hook I'm guessing it'll be called for every page that uses the civihr_default_theme. And you're adding the path to civihr_default_theme to the settings? Sorry, I'm just not familiar with this so don't know what's happening.

Reply:
Here I am adding the path to the theme in a JS variable, because using the same I am adding images to the DOM at 
https://github.com/compucorp/civihr-employee-portal-theme/blob/a7502d444dd04e7ddd7e95b920b912c1dc5f8801/civihr_default_theme/assets/js/civihr_default_theme.script.js#L152